pkg/config/config.go

```diff
@@ -303,18 +349,20 @@ func (c *Config) Write(ctx context.Context, w io.Writer) error {
 		return err
 	}
 
-	for _, context := range contexts {
+	for i, context := range contexts {
+		status := statuses[i]
 		current := ""
-		if context.current {
+		if status.current {
 			current = greenStar
 		}
 
-		_, err = fmt.Fprintf(tb, "%s\t%s\t%s\t%s\t%s\n",
+		_, err = fmt.Fprintf(tb, "%s\t%s\t%s\t%s\t%s\t%s\n",
 			current,
 			context.Name,
 			context.URL,
-			context.account,
-			context.status,
+			status.account,
+			status.user,
```

Comment:
i just added the username to the output, without really asking product or anyone else. it was handy for me, but i should ask them before actually making this change.